api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantServiceIntegrationTests.java

```diff
@@ -241,6 +249,36 @@ void shouldReturnDuplicateStatusWhenEligibleClaimAlreadyExistsForDwpHouseholdIde
         wiremockManager.assertThatEligibilityRequestMade();
     }
 
+    @Test
+    void shouldReturnDuplicateStatusWhenTwoEligibleClaimsForSameNino() throws JsonProcessingException, InterruptedException {
```

Comment:
Minor, but suggest a slight change in wording: `shouldReturnDuplicateStatusWhenTwoSimultaneousClaimsForSameNino`